components/script/dom/htmlformelement.rs

```diff
@@ -791,7 +791,11 @@ impl HTMLFormElement {
 
         let global = self.global();
 
-        let request_body = bytes.extract(&global).into_net_request_body().0;
+        let request_body = bytes
+            .extract(&global)
+            .expect("Couldn't extract body.")
```

Comment:
This seems like we should be handling the error instead of panicking.

Reply:
At this point the `impl Extractable for Vec<u8>` cannot fail, although it returns a result like the others.